src/nomad_material_processing/general.py

```diff
@@ -545,24 +572,39 @@ class SubstrateCrystalProperties(CrystalProperties):
             component=ELNComponentEnum.EnumEditQuantity,
         ),
     )
-    orientation = Quantity(
+    hkl_reciprocal = Quantity(
         type=str,
-        description="""
-        Alignment of crystal lattice with respect to a vector normal to the surface
-        specified using Miller indices.
-        """,
+        description="""Reciprocal (or Absolute) Miller indices, (hkl),
```

Comment:
As discussed, there was some misunderstanding here: you can remove the term "absolute".
Just know that there is a notation `{hkl}` for symmetry-reduced planes.

Reply:
The proper term is "equivalent planes"

Reply:
it's not anymore in the code, I think this is some outdated thing